covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -33,7 +33,7 @@ def home():
 
     @server.route("/covid19/normal")
     def covid19_redirect():
-        return redirect(url_for("covid19_predictions"))
+        return redirect(url_for("covid19_all_predictions"))
```

Comment:
I think single predictions should be the default (i.e. the one that does not lag).

We should also redirect from `/covid19/predictions` now. (Maybe you can even have two `@server.route` decorators for one function?)

Reply:
I disagree, the old URL should go to the dashboard with the old functionality - "Do not break users" :smile: Yes, we might eat too much of their memory, but this consistency seems to be more important to them (the few that I talked to).

`/covid19/predictions/` can go to `/single/`, though.

Reply:
Ok then either is good. Your call for the default for the `/covid19/predictions/`.